src/tests/controllers/badgeBot.test.ts

```diff
@@ -103,10 +103,10 @@ export async function badgeBotTest(t, index1, index2, index3) {
   await sleep(10000)
 
   // CHECK IF NODE2 ACTUALLY RECIEVED THE BAGDE ON THE ELEMENT SERVER
-  const confirm = await confirmBadge(node2, earnBadge.response.id)
-  t.true(confirm, 'Node 2 should recieve the earner badge')
+  // const confirm = await confirmBadge(node2, earnBadge.response.id)
+  // t.true(confirm, 'Node 2 should recieve the earner badge')
 
-  await sleep(1000)
+  // await sleep(1000)
```

Comment:
Are these tests failing now?

Reply:
nvm saw this [ee7d407](https://github.com/stakwork/sphinx-relay/pull/580/commits/ee7d407656693b3956acc35c345c1db5071b83ca)

Are we able to update the tests to match the new flow?

Reply:
Yes, I would be able to do that but not on this PR, because I would have to change the flow of how the badge bot works a little...

When I have that, I would update the test

Reply:
ok sounds good, merging